packages/ckb-sdk-core/src/index.ts

```diff
@@ -72,29 +80,35 @@ class Core {
       codeHashIndex,
     })
 
-  public loadSystemCell = async () => {
+  public loadSecp256k1Dep = async () => {
+    /**
+     * cell list
+     * @link https://github.com/nervosnetwork/ckb/blob/dbadf484cea6bdba0329d58102726068be997a50/docs/hashes.toml
+     */
     const block = await this.rpc.getBlockByNumber('0')
     if (!block) throw new Error('Cannot load the genesis block')
-    const cellTx = block.transactions[0]
-    if (!cellTx) throw new Error('Cannot load the transaction which has the system cell')
-    const cell = cellTx.outputs[1]
-    if (!cell) throw new Error('Cannot load the system cell')
-
-    const s = this.utils.blake2b(32, null, null, this.utils.PERSONAL)
-    s.update(this.utils.hexToBytes(cell.data.replace(/^0x/, '')))
-    const codeHash = s.digest('hex')
-    const outPoint = {
-      blockHash: block.header.hash.replace(/^0x/, ''),
-      cell: {
-        txHash: cellTx.hash.replace(/^0x/, ''),
-        index: '1',
-      },
+    const secp256k1CodeTx = block.transactions[0]
+    if (!secp256k1CodeTx) throw new Error('Cannot load the transaction which has the secp256k1 code cell')
+    if (!secp256k1CodeTx.outputs[1]) {
+      throw new Error('Cannot load the secp256k1 code because the specific output is not found')
     }
-    this.config.systemCellInfo = {
-      codeHash,
-      outPoint,
+
+    const secp256k1CodeHash = this.utils.blake160(secp256k1CodeTx.outputsData[1], 'hex')
+
+    const secp256k1DepTx = block.transactions[1]
+    if (!secp256k1DepTx) throw new Error('Cannot load the transaction which has the secp256k1 dep cell')
+    if (!secp256k1DepTx.outputs[0]) {
+      throw new Error('Cannot load the secp256k1 dep because the specific output is not found')
+    }
+
+    this.config.secp256k1Dep = {
+      codeHash: secp256k1CodeHash,
+      outPoint: {
+        txHash: secp256k1DepTx.hash.replace(/^0x/, ''),
```

Comment:
Would it be better to use a uniform format of `hash` with ckb?

Reply:
I'd like to commit another PR for the universal format.